huxley/settings/main.py

```diff
@@ -145,3 +145,10 @@
 
 # Celery Configuration Options
 CELERY_RESULT_BACKEND = 'django-db'
+
```

Comment:
remove these keys and set them to null for the code base, we will set private keys in prod

Reply:
Done

Reply:
Not that its really going to happen but if the keys were already in the git history, you may want to regenerate the keys on Inuit's side.